pkg/container/resolver_test.go

```diff
@@ -104,8 +111,88 @@ func TestResolverFail(t *testing.T) {
 		common.ToPtr(""),
 		common.ToPtr(false),
 		false,
+		nil,
 	})
 	specs, err = resolver.Finish()
 	assert.Error(t, err)
 	assert.Len(t, specs, 0)
 }
+
+func TestResolverLocalManifest(t *testing.T) {
+	currentUser, err := user.Current()
+	if err != nil {
```

Comment:
Why not just `assert.NoError(t, err)` here? It seems to be the style of the other tests and is slightly shorter :) (here and in some places below)